lib/src/lib/tokenizer/tokenizer.ts

```diff
@@ -49,10 +49,10 @@ export abstract class Tokenizer {
   public tokenizeWithMapping(text: string): [string[], Region[]] {
     const resultTokens: Array<string> = [];
     const positionMapping: Array<Region> = [];
-    for (const { token, location } of this.generateTokens(text)) {
+    this.generateTokens(text).forEach(({ location, token }) => {
       resultTokens.push(token);
```

Comment:
I personally prefer keeping the `for` loop instead of a `forEach`. This is consistent with the other loops we are using in the `lib` and `core` modules of Dolos.

Reply:
Not sure if it matters here, but in addition, forEach is up to 3 times slower compared to a simple for.